### PR TITLE
Create file folders for better organization

### PR DESCRIPTION
**Pull request by reuterShawn**
Created the following folders called components, components-test, services,  and pages. We also could have a folder containing all of the interfaces if necessary.

### CODE REVIEW

**Comment by reuterShawn** [comment]
Here is a [link ](https://docs.google.com/document/d/1OOHVjGm2vfJmpZhN98ZCAutcOq8bocC1T1lUbWkHr7s/edit )to the google doc with the file structure. We could also make a folder for all of the files with  an interface if we feel it is necessary.

File: angular.json [hunk 1]
```diff
@@ -148,5 +148,8 @@
       }
     }
   },
-  "defaultProject": "buzz-about"
-}
+  "defaultProject": "buzz-about",
+  "cli": {
+    "analytics": false
+  }
```
Comment (helloworld12321) [inline angular.json:154]: Sounds good!

File: angular.json [hunk 2]
```diff
@@ -148,5 +148,8 @@
       }
     }
   },
-  "defaultProject": "buzz-about"
-}
+  "defaultProject": "buzz-about",
+  "cli": {
+    "analytics": false
```
Comment (floogulinc) [inline angular.json:153]: I don't think we should set this for the entire project. It should be an individual choice.

Comment (reuterShawn) [reply angular.json:153]: Sorry Paul I didn't mean to push that up there was 100 files that were being committed so I did not notice that this was one of them.

File: angular.json (hunk 1 above)
Comment (floogulinc) [inline angular.json:154]:
```suggestion
  "defaultProject": "buzz-about"
```

**Pull request merged**